### PR TITLE
coverage: fix description text

### PR DESCRIPTION
**Pull request by Jaehyuk-Lee**
with a simple translation fix

### CODE REVIEW

File: util/coverage/coverage.js [hunk 1]
```diff
@@ -370,7 +370,7 @@ document.addEventListener('DOMContentLoaded', () => {
   buildLanguageSelect(languageSelect, lang);
 
   const description = document.getElementById('description-text');
-  description.innerText = translate(miscStrings.description, lang);
+  description.innerHTML = translate(miscStrings.description, lang);
```
[the remark this answers is not in the document]
Comment (quisquous) [reply util/coverage/coverage.js:374]: `innerText` can only be text.  If there's emoji or html, it has to be `innerHTML`.

[the remark this answers is not in the document]
Comment (Trim21) [reply util/coverage/coverage.js:374]: Oh yes I find that HTML.

[the remark this answers is not in the document]
Comment (Jaehyuk-Lee) [reply util/coverage/coverage.js:374]: Emoji is inserted directly from the HTML file, so doesn't matter.

[the remark this answers is not in the document]
Comment (Jaehyuk-Lee) [reply util/coverage/coverage.js:374]: When you insert emoji itself directly like 📢, it's ok with innerText.
When you insert emoji with unicode like &#x1F4E2, it doesn't work.

**Pull request merged**